content/en/docs/how-to/plugin-management.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Install and manage notation plugins
+description: Install and manage notation plugins
+weight: 9
+---
+
+Plugins for `notation` provide additional functionality and integration. For example, the [notation-azure-kv](https://github.com/Azure/notation-azure-kv) plugin allows you to sign a Notation-generated payload with a private key in [Azure Key Vault](https://learn.microsoft.com/en-us/azure/key-vault/general/overview?wt.mc_id=azurelearn_inproduct_oss_notaryproject).
+
+## Install a plugin
+
+To install a plugin, download the plugin binary, create a `{plugin-name}` directory in `{NOTATION_LIBEXEC}/plugins/`, and add the plugin binary to that directory.
+
+{{% alert title="Warning" color="warning" %}}
+Before creating the `{plugin-name}` directory, confirm you are using a name that follows the [naming structure](#plugin-naming-structure) for plugins. Plugins that do not follow the naming structure will not be recognized by `notation`.
+{{% /alert %}}
+
+The following example downloads and installs version 0.6.0 of [notation-azure-kv](https://github.com/Azure/notation-azure-kv) plugin for on macOS with Apple Silicon using the default location for the Notation installation.
+
+{{% alert title="Warning" color="warning" %}}
+The following example only works for version 0.6.0of the *notation-azure-kv* plugin on macOS with Apple Silicon using the default location for the Notation installation. You will need to update the filenames, location, and commands for other plugins, versions, and platforms. For more details on the default location of that directory on each platform, see [Notation directory structure for system configuration]({{< ref "/docs/concepts/directory-structure" >}}).
+{{% /alert %}}
+
+```console
+curl -Lo notation-azure-kv_0.6.0_darwin_arm64.tar.gz "https://github.com/Azure/notation-azure-kv/releases/download/v0.6.0/notation-azure-kv_0.6.0_darwin_arm64.tar.gz"
+mkdir -p ~/Library/Application\ Support/notation/plugins/azure-kv
+tar xvzf notation-azure-kv_0.6.0_darwin_arm64.tar.gz -C ~/Library/Application\ Support/notation/plugins/azure-kv notation-azure-kv
+```
+
+To confirm you plugin is installed, run `notation plugin list`. For example:
+
+```console
+notation plugin list
+```
+
+Confirm the plugin is listed in the output. For example:
+
+```console
+$ notation plugin list
+NAME       DESCRIPTION                                   VERSION   CAPABILITIES                ERROR
+azure-kv   Sign artifacts with keys in Azure Key Vault   0.6.0     [SIGNATURE_GENERATOR.RAW]   <nil>
+```
+
+## Uninstall a plugin
+
+To remove a plugin, delete the entire `{NOTATION_LIBEXEC}/plugins/{plugin-name}/` directory.
+
+To confirm you plugin is uninstalled, run `notation plugin list`. For example:
+
+```console
+notation plugin list
+```
+
```

Comment:
For security considerations, could you also add a Note box to remind users to remove the related key from the signing key list using `notation key delete`?

Reply:
@FeynmanZhou this seems like advice that only applies to the KMS plugin so it doesn't belong in the general plugin management doc. Maybe we can add a note here: https://github.com/Azure/notation-azure-kv#getting-started?